Buildings/Air/Systems/SingleZone/VAV/BaseClasses/HysteresisWithHold.mo

```diff
@@ -0,0 +1,146 @@
+    annotation (Line(points={{100.5,0},{116,0}}, color={0,0,0}));
+  connect(toTrue1.outPort, outputTrue.inPort[2]) annotation (Line(points={{21.5,
+          90},{46,90},{70,90},{70,-0.5},{79,-0.5}}, color={0,0,0}));
+  connect(toFalse.outPort, outputFalse.inPort[2]) annotation (Line(points={{121.5,
+          0},{140,0},{140,20},{12,20},{12,-0.5},{19,-0.5}}, color={0,0,0}));
+  connect(timer.y, greaterEqualThreshold.u)
+    annotation (Line(points={{-39,-50},{-22,-50}}, color={0,0,127}));
+  connect(greaterEqualThreshold.y, and1.u1)
+    annotation (Line(points={{1,-50},{18,-50}}, color={255,0,255}));
+  connect(hysteresis.y, and1.u2) annotation (Line(points={{-59,0},{-50,0},{-50,-16},
+          {-80,-16},{-80,-80},{10,-80},{10,-58},{18,-58}}, color={255,0,255}));
+  connect(outputFalse.active, timer.u) annotation (Line(points={{30,-11},{30,-20},
+          {-76,-20},{-76,-50},{-62,-50}}, color={255,0,255}));
+  connect(and1.y, toTrue.condition) annotation (Line(points={{41,-50},{50,-50},{
+          60,-50},{60,-12}}, color={255,0,255}));
+  connect(outputTrue.active, timer1.u) annotation (Line(points={{90,-11},{90,-11},
+          {90,-20},{64,-20},{64,-50},{78,-50}}, color={255,0,255}));
+  connect(timer1.y, greaterEqualThreshold1.u)
+    annotation (Line(points={{101,-50},{118,-50}}, color={0,0,127}));
+  connect(greaterEqualThreshold1.y, and2.u1)
+    annotation (Line(points={{141,-50},{158,-50}}, color={255,0,255}));
+  connect(not1.y, and2.u2) annotation (Line(points={{-19,50},{-19,50},{200,50},{
+          200,-80},{150,-80},{150,-58},{158,-58}}, color={255,0,255}));
+  connect(and2.y, toFalse.condition) annotation (Line(points={{181,-50},{190,-50},
+          {190,-24},{120,-24},{120,-12}}, color={255,0,255}));
+  connect(outputTrue.active, on) annotation (Line(points={{90,-11},{90,-11},{90,
+          -20},{180,-20},{180,0},{230,0}}, color={255,0,255}));
+  annotation (defaultComponentName="hysWitHol",
+Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-100,-100},{220,120}})),
+Icon(   graphics={
+        Text(
+          extent={{-66,-40},{62,-82}},
+          lineColor={0,0,0},
+          textString="%uLow     %uHigh"),
+          Polygon(
+            points={{-12,8},{0,2},{-12,-4},{-12,8}},
+            lineColor={0,0,0},
+            fillColor={0,0,0},
+            fillPattern=FillPattern.Solid),
+          Line(points={{-44,2},{-12,2}}),
+          Line(points={{0,34},{0,-36}}),
+          Line(points={{0,2},{36,2}}),
+          Polygon(
+            points={{36,8},{48,2},{36,-4},{36,8}},
+            lineColor={0,0,0},
+            fillColor={0,0,0},
+            fillPattern=FillPattern.Solid),
+          Rectangle(extent={{48,18},{86,-16}}, lineColor={0,0,0}),
+          Rectangle(extent={{-82,18},{-44,-16}}, lineColor={0,0,0})}),
+Documentation(info="<html>
+<p>
+Model for a hysteresis block that optionally allows to specify a hold time. 
+During the hold time, the new state is hold.
```

Comment:
Can you please expand so that it is clear what the model does without having to look at the Modelica implementation. (This will also be needed for OpenBuildingControl.)